handlers/handlers.go

```diff
@@ -194,10 +195,131 @@ func (s *ServiceSet) serviceHTML(db *mbtiles.DB) handlerFunc {
 	}
 }
 
+type tileCoord struct {
```

Comment:
What about 

```
type Tile struct {
    z uint8
    x, y uint64
    ext string
}
```

Which then gives us a unique identifier structure for a given tile, and allows us to capture extension here.

Reply:
@fawick It does not need to be exported, I wasn't thinking properly.

Long term, I'm thinking it might make sense to store multiple formats in a given database, though this isn't entirely to spec.  E.g., a PNG (primary) version of the tile, and a WEBP (secondary) version of the tile for supported browsers.  (in part, that particular choice stems from sub-optimal results so far with Go's `webp` encoder and doing re-encoding on the fly).

If it makes more sense to you to leave `ext` out, that is fine.